### ISSUE TITLE
Duplicate loaders on the modded runtime classpath

### ISSUE THREAD

**Issue by ghost** [QuiltMC/quilt-loom#52]
# The Issue
Basically it's all the in the title, **Quilt Loom** adds **Fabric Loader** or **Quilt Loader** to the `modRuntimeClasspathMainMapped` configuration twice, unsure as to the root cause however if you use **Quilt Loader** this is kind of a non-issue due to it just ignoring one of the entries.
> [14:37:57] [main/WARN] (Quilt Loader/Solving) Ignoring duplicate mod quilt_loader of the same version 0.26.3 loaded from C:\Users\Quinn Semele\Projects\Modding\mod issues\.gradle\quilt-loom-cache\remapped_mods\org_quiltmc_quilt_mappings_1_21_1_21_build_18_v2\org\quiltmc\quilt-loader\0.26.3\quilt-loader-0.26.3.jar

However if your using **Fabric Loader** in a project then trying to launch the game will fail.
> Caused by: java.lang.IllegalStateException: duplicate fabric loader classes found on classpath: jar:file:/C:/Users/Quinn Semele/.gradle/caches/modules-2/files-2.1/net.fabricmc/fabric-loader/0.15.11/9c2be53d84ec5c4fb58bb5b8d9c8a629047c9ee9/fabric-loader-0.15.11.jar!/net/fabricmc/loader/api/FabricLoader.class, jar:file:/C:/Users/Quinn Semele/Projects/Modding/mod issues/.gradle/quilt-loom-cache/remapped_mods/org_quiltmc_quilt_mappings_1_21_1_21_build_18_v2/net/fabricmc/fabric-loader/0.15.11/fabric-loader-0.15.11.jar!/net/fabricmc/loader/api/FabricLoader.class
	at net.fabricmc.loader.impl.util.LoaderUtil.verifyClasspath(LoaderUtil.java:65)

# The Impact
Basically I'm a multi-loader dev and I've been trying to switch away from **Architectury** to native build tools (**NeoForge's ModDevGradle** and **Quilt's Loom**) and this issue is blocking me from testing the **Fabric** release of my mod, and if I switch to **Fabric's Loom** well then the **Quilt** release has a few issues too and so basically this being fixed would be great ^_^

---
Oh I've also made a repo showing off this issue if you think it might aid with debugging efforts: https://github.com/quinn-semele/mod-issues

**Comment by ghost** [comment]
Did some testing, broke between `1.2.3` and `1.3.1`

Seems like the only noteable commit within `1.3.x` is https://github.com/QuiltMC/quilt-loom/commit/4f7bccd45653516e6eb4804c06304e1539dbb1ad
